spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -10,16 +10,6 @@
   before do
     load File.expand_path('prepare_testapp.rb', __dir__)
   end
-
-  let(:charset_alter_table) do
-    if defined?(Mysql2)
-      <<~EOS
-
-
-        execute "ALTER TABLE `adverts` CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
-      EOS
-    end
-  end
```

Comment:
@jebentier While I was here, I put the CHARACTER SET and COLLATE directly in the `CREATE TABLE` command. The ALTER is now only used when you're changing it.